### ISSUE TITLE
Support for ng-docs?

### ISSUE THREAD

**Issue by umayr** [umayr/generator-ng-bat#5]
Should there be support for [ng-docs](https://github.com/angular/angular.js/wiki/Writing-AngularJS-Documentation)?

If there is, it should be conditional. For e.g.
```
   $ yo ng-bat --include-docs
```

**Comment by mohuk** [comment]
Nope, docs should be optional as they only make sense for services not for
controllers
